osu.Game/Overlays/SettingsOverlay.cs

```diff
@@ -93,7 +96,11 @@ private void load(OsuGame game)
                         new SidebarButton
                         {
                             Section = section,
-                            Action = b => sectionsContainer.ScrollContainer.ScrollTo(b),
+                            Action = s =>
+                            {
+                                sectionsContainer.ScrollTo(s);
+                                sidebar.State = ExpandedState.Contracted;

```

Comment:
I could be hovering down the list to get to another item than where my mouse is positioned - this is how I initially noticed this in the first place.

I don't want to be too stuck up on this but I just don't see the point given how settings acts in general (pushing the settings out, contracting them back in) is all.